packages/frontend/src/pages/Space.tsx

```diff
@@ -63,16 +63,14 @@ const Space: FC = () => {
     const { user, health } = useApp();
     const { track } = useTracking();
 
-    const canCreateNestedSpaces = useMemo(() => {
-        const userCanManageSpace = user.data?.ability?.can(
+    const userCanManageSpace = useMemo(() => {
+        return user.data?.ability?.can(
```

Comment:
does this even require useMemo? it's just a function call returning a boolean

Reply:
no it was like that before. I'll update. thanks!